src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 2 * $calendar-padding $calendar-padding;
+  margin: auto;
+  width: $calendar-width;
+
+  &__day {
+    display: flex;
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid #000;
+
+    justify-content: center;
+    align-items: center;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: background-color 500ms, transform 500ms;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &__day:nth-child(#{$index})::before {
+      content: "#{$index}";
+    }
+  }
+
+  @each $first-day, $margin in $week {
+    &--start#{$first-day} :first-child {
```

```suggestion
    &--start-day--#{$first-day} :first-child {
```
створюється не той клас

---

Спасибо, поправил